src/components/DatePickerFullScreen.js

```diff
@@ -0,0 +1,518 @@
+          styles.selectedDateWrapper,
+          this.props.selectedDateWrapperStyle
+        ]}>
+          {this._renderSelectedDate()}
+        </div>
+        <div className='mx-date-picker-full-screen-calendar-scrim' key='calendarModal' style={[
+          styles.calendarModal,
+          this.state.showCalendar && styles.calendarShow,
+          this.props.calendarScrimFixedPosition && { position: 'fixed' }
+        ]}>
+          <div style={styles.closeIcon}>
+            <Icon
+              onClick={this._handleCloseClick.bind(this)}
+              size='32px'
+              type='close'
+            />
+          </div>
+          <div className='mx-date-picker-full-screen-calendar-wrapper' style={styles.calendarWrapper}>
+            {this._renderTitle(styles)}
+            <div className='mx-date-picker-full-screen-calendar-header' key='calendarHeader' style={[styles.calendarHeader, { borderBottomStyle: this.props.showDayBorders ? 'solid' : 'none' }, styles.clearFix]}>
+              <Icon
+                onClick={this._handlePreviousClick.bind(this)}
+                size='32px'
+                style={[styles.navIcon, styles.navLeft, this.props.showDayBorders && styles.borderRight]}
+                type='caret-left'
+              />
+              {currentDate.format('MMMM YYYY')}
+              <Icon
+                onClick={this._handleNextClick.bind(this)}
+                size='32px'
+                style={[styles.navIcon, styles.navRight, this.props.showDayBorders && styles.borderLeft]}
+                type='caret-right'
+              />
+            </div>
+            <div style={styles.calendarContainer}>
+              {this._renderMonthTable(currentDate, selectedDate)}
+            </div>
+            <div style={styles.clearFix}></div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+DatePickerFullScreen.propTypes = {
+  calendarScrimFixedPosition: React.PropTypes.bool,
+  caretWrapperStyle: React.PropTypes.object,
+  closeOnDateSelect: React.PropTypes.bool,
+  defaultDate: React.PropTypes.number,
+  format: React.PropTypes.string,
+  inputStyle: React.PropTypes.object,
+  locale: React.PropTypes.string,
+  minimumDate: React.PropTypes.number,
+  onDateSelect: React.PropTypes.func,
+  placeholderText: React.PropTypes.string,
+  placeholderTextStyle: React.PropTypes.object,
+  selectedDateWrapperStyle: React.PropTypes.object,
+  showCaret: React.PropTypes.bool,
+  showDayBorders: React.PropTypes.bool,
+  style: React.PropTypes.object,
+  title: React.PropTypes.string,
+  useInputForSelectedDate: React.PropTypes.bool
+};
+
+DatePickerFullScreen.defaultProps = {
+  calendarScrimFixedPosition: false,
+  closeOnDateSelect: false,
+  format: 'MMM D, YYYY',
+  locale: 'en',
+  onDateSelect () {},
+  showCaret: true,
+  showDayBorders: false,
+  title: 'Select A Date',
+  useInputForSelectedDate: true
+};
+
+const styles = {
+  caret: {
+    color: '#CCCCCC',
```

Comment:
Could we update all the colors to be pulled from the Style constants file? Refer to frontify for the color pallet to use.